code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -315,3 +315,25 @@
 
 /mob/living/simple_animal/mouse/plague
 	disease_carrier = 1
+
+/mob/living/simple_animal/mouse/mouse_op
+	name = "mouse operative"
+	namenumbers = FALSE
+	min_oxy = 0
+	minbodytemp = 0
+	maxbodytemp = 5000
+	maxHealth = 50
+	health = 50
+	universal_speak = 1
+	can_chew_wires = 1
+
+/mob/living/simple_animal/mouse/mouse_op/New()
+	..()
+	desc = "Oh no..."
```

Comment:
why are these three lines in New() rather than in the type definition?

Reply:
It probably doesn't _need_ to be done that way, but it's because I call the parent New() since it sets a bunch of default fallback icons.